core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.ts

```diff
@@ -444,13 +444,15 @@ export class ContentTypeFieldsDropZoneComponent implements OnInit, OnChanges, On
     }
 
     private setUpDragulaScroll(): void {
-        const drake = this.dragulaService.find('fields-bag')?.drake;
+        // const drake = this.dragulaService.find('fields-bag')?.drake;
         autoScroll([this.elRef.nativeElement.parentElement], {
             margin: 100,
             maxSpeed: 60,
             scrollWhenOutside: true,
             autoScroll() {
-                return this.down && drake.dragging;
+                // TODO: Verify if this works
+                // return this.down && drake.dragging;
+                return this.down;
```

Comment:
This doesn't look good... did you try this?

Reply:
Fixed and tested.